packages/libraries/router/src/main.rs

```diff
@@ -13,7 +11,7 @@ fn main() {
     register();
 
     // Initialize the Hive Registry and start the Apollo Router
-    match HiveRegistry::new(None).and(apollo_router::main()) {
+    match HiveRegistry::setup(None).and(apollo_router::main()) {
```

Comment:
This will be a breaking change, no?

Reply:
https://the-guild.dev/graphql/hive/docs/other-integrations/apollo-router
and click on "Custom" tab to see.

If somebody wants to use Hive Registry in their Apollo Router compiled from source, then they do use `::new`.

Reply:
I see, that's unfortunate 🥲 fixed in https://github.com/graphql-hive/console/pull/5632/commits/30822babe706642f0d73759e643fa4897cb5daa5